test/node/helpers.js

```diff
@@ -1,12 +1,12 @@
-var { expect } = require('chai');
-var constants = require('../../src/core/constants.js');
+import expect from 'chai';
```

Comment:
we have to import `expect` like this:
```suggestion
import { expect } from 'chai';
```